src/pudl/package_data/eia923/column_maps/financial_information.csv

```diff
@@ -0,0 +1,31 @@
+fly_ash_revenues_thousand_dollars,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues_thousand_dollars,fly_ash_revenues_thousand_dollars,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues,fly_ash_revenues
+bottom_ash_revenues_thousand_dollars,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues_thousand_dollars,bottom_ash_revenues_thousand_dollars,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues,bottom_ash_revenues
+fly_bottom_ash_intermingled_revenues_thousand_dollars,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues_thousand_dollars,fly_bottom_ash_intermingled_revenues_thousand_dollars,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues,fly_bottom_ash_intermingled_revenues
+fgd_byproducts_revenues_thousand_dollars,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues_thousand_dollars,fgd_byproducts_revenues_thousand_dollars,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues
+other_byproducts_revenues_thousand_dollars,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues_thousand_dollars,other_byproducts_revenues_thousand_dollars,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues
+total_revenues_thousand_dollars,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues_thousand_dollars,total_revenues_thousand_dollars,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues
```

Comment:
Similar to another comment above, maybe we should make this column more specific to the table. Something like `total_byproduct_revenue` (taken from the layout tab in the raw file `EIA923_Schedule_8_Annual_Environmental_Information_2022_Final`).